source/includes/_one-time-sync-individual-custom-check.html.md.erb

```diff
@@ -0,0 +1,491 @@
+    </tr>
+    <tr>
+      <td><code>documents_metadata</code></td>
+      <td>Object</td>
+      <td>No</td>
+      <td>Any additional identity numbers that individual has. Note that the individual''s national identity number(s) is stored in <code>personal_details.national_identity_number</code>
+      </td>
+    </tr>
+    <tr>
+      <td><code>customer_refs</code></td>
+      <td>String</td>
+      <td>Yes</td>
+      <td>
+        A label used by Passfort customers to identify the profile the check is being run against. 
+      </td>
+    </tr>
+    <tr>
+      <td><code>address_history</code></td>
+      <td>Object</td>
+      <td>No</td>
+      <td>
+        A list of addresses in which the individual has lived. Each of these can have an approximate start and end date. Refer to <a href="https://developer.passfort.com/api#tag/Profiles/paths/~1profiles~1%7Bprofile_id%7D~1collected_data/post">Passfort API Documentation</a> for additional details.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+
+
+### Response fields
+
+<table>
+  <thead>
+    <th>Field</th>
+    <th>Type</th>
+    <th>Required?</th>
+    <th>Description</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>errors</code></td>
+      <td>
+        An array containing one or more
+        <a href="#check-errors">Errors</a>
+      </td>
+      <td>No</td>
+      <td>
+        A list of errors that occurred while running this check. If this is
+        provided and it''s not empty, the result of this check will be
+        considered to be errored by PassFort, regardless of the other contents
+        of this response.
+      </td>
+    </tr>
+    <tr>
+      <td><code>warnings</code></td>
+      <td>
+        An array containing one or more warnings
+      </td>
+      <td>No</td>
+      <td>
+        Essentially the same as <code>errors</code>, except the
+        <code>sub_type</code> and <code>data</code> fields are not present, and
+        providing warnings will not cause PassFort to consider the check
+        as errored.
+      </td>
+    </tr>
+    <tr>
+      <td><code>provider_data</code></td>
+      <td>
+        Any valid JSON value
+      </td>
+      <td>No</td>
+      <td>
+        This should be the structured JSON data returned by the data provider,
+        or a conversion of the data provider''s response, to allow
+        PassFort to investigate issues that may arise with your integration.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+>This endpoint should return JSON in the following format:
+
+```json
+<%= pretty_json("responses/run_check_sync_individual_custom", "json") %>
+```
+#### `check_output`
+
+<table>
+  <thead>
+    <th>Field</th>
+    <th>Type</th>
+    <th>Required?</th>
+    <th>Description</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>decision</code></td>
+      <td><code>DecisionClass</code></td>
+      <td>Yes</td>
+      <td>
+        The check will return one of the following decisions: "PASS | FAIL | PARTIAL | WARN | ERROR"
```

Comment:
Should the decisions be formatted as <code>?

Reply:
"will return" to "returns"

Reply:
Put each decision in its own set of quotation marks and use commas to separate:
"PASS", "FAIL", "PARTIAL", "WARN", "ERROR"